benchmark_runner/common/template_operations/template_operations.py

```diff
@@ -162,14 +162,14 @@ def __generate_yamls_internal(self, scale: str = None, scale_num: str = None, sc
             answer[filename] = f"{template.render(render_data)}\n"
             # namespace
             if os.path.isfile(os.path.join(self.__dir_path, f'namespace_template.yaml')):
-                answer['namespace.yaml'] = render_yaml_file(dir_path=self.__dir_path, yaml_file='namespace_template.yaml', environment_variable_dict=self.__environment_variables_dict)
-            # windows workload
+                answer['namespace.yaml'] = render_yaml_file(dir_path=self.__dir_path, yaml_file='namespace_template.yaml', environment_variable_dict=self.__environment_variables_dict) + '\n'
```

Comment:
You should figure out why the trailing newline is not correctly appended by render_yaml_file.